CRM-21416: Add reset link beside 'Search' button below Advance Search form

monishdeb opened the pull request:
Overview
----------------------------------------
Add 'Reset Form' link beside 'Search' button below Advance Search form

Before
----------------------------------------
![screen shot 2017-11-09 at 1 38 48 pm](https://user-images.githubusercontent.com/3735621/32594955-012ecdf0-c554-11e7-996a-d2cfa0245985.png)

After
----------------------------------------
![screen shot 2017-11-09 at 1 37 31 pm](https://user-images.githubusercontent.com/3735621/32594963-09a69fbc-c554-11e7-9c07-85d9c323b3c9.png)

---

 * [CRM-21416: Add reset link beside 'Search' button below Advance Search form](https://issues.civicrm.org/jira/browse/CRM-21416)

mattwire commented:
@monishdeb Have tested this and it works fine.  It's a trivial template change that will have no impact elsewhere.  It would be nice to implement this on some of the other search forms too, but that shouldn't block this PR.
@eileenmcnaughton @colemanw ready to merge?

colemanw commented:
I'm confused about one thing @monishdeb - why is the markup set to both float left *and* float right?

monishdeb commented:
@colemanw thanks for pointing that out. Earlier I have added float left to place the link next to Search  button, but didn't knw that ```crm_right``` class was the reason to align the link at right. I have removed both the style and class and updated the PR. Tested on different theme, works for me:
![screen shot 2017-11-10 at 12 16 16 pm](https://user-images.githubusercontent.com/3735621/32646196-7073d864-c611-11e7-8aa4-4080aa4e85fc.png)

![screen shot 2017-11-10 at 12 15 41 pm](https://user-images.githubusercontent.com/3735621/32646205-779ff38e-c611-11e7-9412-20f0346a4a03.png)
